### PR TITLE
Document config.assets.gzip flag.

### PR DESCRIPTION
**Pull request by vipulnsward**
Follow up of https://github.com/rails/sprockets-rails/pull/342

r? @schneems

### CODE REVIEW

**Comment by schneems** [comment]
Thanks! ❤️

File: guides/source/asset_pipeline.md [hunk 1]
```diff
@@ -1111,6 +1111,17 @@ effect on the application. Instead, setting `config.assets.css_compressor` and
 `config.assets.js_compressor` will control compression of CSS and JavaScript
 assets.
 
+
+### Serving GZipped version of assets
+
+By default, gzipped version of compiled assets will be generated, along 
+with the non-gzipped version of assets. Gzipped assets help reduce, the transmission of 
+date over the wire. You can configure this by setting the `gzip` flag.
```
Comment (connorshea) [inline guides/source/asset_pipeline.md]: There shouldn't be a comma after "reduce", and "date" should be "data", presumably?

Glad this is documented now, though :D

Comment (vipulnsward) [inline guides/source/asset_pipeline.md]: The comma should exist, but the typo should go away. Send in a PR to fix :D

Comment (schneems) [inline guides/source/asset_pipeline.md]: More commits for everyone!!!!!!!!! 🎉 🎉 🎉

**Pull request merged**